react/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,17 @@
+import styles from './ButtonGroup.less';
+import React, { PropTypes } from 'react';
+
+export default function ButtonGroup({ children }) {
+  return (
+    <div>{children.map(child => React.cloneElement(child, { className: styles.button }))}</div>
```

Comment:
Yes true but the only thing this component does right now is add margin so is this necessarily a bad thing?